src/pages/docs/auth/capabilities.mdx

```diff
@@ -11,82 +11,83 @@ languages:
+* A resource of `foo*` (without a colon) will only match the single channel literally called `foo*`.
 
-A resource can also be a queue, in which case it will start with @[queue]@, for example @[queue]appid-queuename@. This is unambiguous as channel names may not begin with a @[@. Similar wildcard rules apply, for example @[queue]*@ will match all queues.
+A resource can also be a queue, in which case it will start with `[queue]`, for example `[queue]appid-queuename`. This is unambiguous as channel names may not begin with a `[`. Similar wildcard rules apply, for example `[queue]*` will match all queues.
 
-A resource can also be a metachannel, in which case it will start with @[meta]@, for example @[meta]metaname@. This is unambiguous as channel names may not begin with a @[@.  @[meta]*@ will match all metachannels. Just @*@ on its own will not: it will match all possible normal channels, but no metachannels.
+A resource can also be a metachannel, in which case it will start with `[meta]`, for example `[meta]metaname`. This is unambiguous as channel names may not begin with a `[`.  `[meta]*` will match all metachannels. Just `*` on its own will not: it will match all possible normal channels, but no metachannels.
 
-You can also have a resource name of @[*]*@, which will match all queues, all metachannels, and all channels.
+You can also have a resource name of `[*]*`, which will match all queues, all metachannels, and all channels.
 
-Wildcards are also supported for "operations":#capability-operations, by requesting an operations list of @['*']@.
+Wildcards are also supported for [operations](#capability-operations), by requesting an operations list of `['*']`.
 
-h2(#capability-operations). Capability operations
+## Capability operations <a id="capability-operations"/>
 
 The following capability operations are available for API keys and issued tokens:
 
-- subscribe := can subscribe to messages and presence state change messages on channels, and get the presence set of a channel
-- publish := can publish messages to channels
-- presence := can register presence on a channel (enter, update and leave)
-- object-subscribe := can subscribe to updates to objects on a channel
-- object-publish := can update objects on a channel
-- history := can retrieve message and presence state history on channels
-- stats := can retrieve current and historical usage statistics for an app
-- push-subscribe := can subscribe devices for push notifications
-- push-admin := can manage device registrations and push subscriptions for all devices in an app
-- channel-metadata := can get metadata for a channel, and enumerate channels
-- privileged-headers := can set data in the privileged section of the "message extras":/docs/api/realtime-sdk/messages#extras
+- **subscribe**: can subscribe to messages, objects, and presence state change messages on channels, and get the presence set of a channel
```

Comment:
I think this would be more readable as a table. It's the sad fact of losing definition lists for now.